doc/source/whatsnew/v2.2.0.rst

```diff
@@ -169,6 +169,7 @@ Bug fixes
 ~~~~~~~~~
 - Bug in :class:`AbstractHolidayCalendar` where timezone data was not propagated when computing holiday observances (:issue:`54580`)
 - Bug in :class:`pandas.core.window.Rolling` where duplicate datetimelike indexes are treated as consecutive rather than equal with ``closed='left'`` and ``closed='neither'`` (:issue:`20712`)
+- Bug in testing PyArrow compatibility: Division of duration dtypes is only supported in upcoming arrow 14.0.0 (:issue:`55020`)
```

Comment:
Could you move this to `v2.1.1.rst`?

Reply:
Actually could you remove this whatsnew entirely. We don't typically mention testing fixes

Reply:
Done